ion-schema/src/isl/mod.rs

```diff
@@ -273,14 +331,18 @@ mod isl_tests {
     use crate::isl::IslVersion;
     use crate::isl::*;
     use crate::result::IonSchemaResult;
+    use crate::system::SchemaSystem;
     use ion_rs::element::Element;
     use ion_rs::types::decimal::*;
     use ion_rs::types::integer::Int as IntegerValue;
     use ion_rs::types::timestamp::Timestamp;
-    use ion_rs::IonType;
     use ion_rs::Symbol;
+    use ion_rs::{IonType, TextWriterBuilder};
+    use nom::AsBytes;
```

Comment:
`nom` is a pretty heavy dependency to bring in for [the `AsBytes` trait](https://docs.rs/nom/latest/nom/trait.AsBytes.html), which seems to be a helper for fixed-sized arrays. Consider either:
* Removing the dependency and using [`Vec::as_slice`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.as_slice) to access the bytes of the `Vec<u8>`.
* Downgrading the dependency to a dev dependency.

Reply:
> `nom` is a pretty heavy dependency to bring in for [the `AsBytes` trait](https://docs.rs/nom/latest/nom/trait.AsBytes.html), which seems to be a helper for fixed-sized arrays. Consider either:
> 
>     * Removing the dependency and using [`Vec::as_slice`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.as_slice) to access the bytes of the `Vec<u8>`.
> 
>     * Downgrading the dependency to a dev dependency.

According to your suggestion and since I am using this only for tests, I will remove the dependency and just use `as_slice`.